The Sprite Changes

wretlaw120 opened the pull request:
Made new sprites for the three lowest tier of nuclear artillery shell (20t, 500t, and 1kt) and all nuclear rockets. Cleaned up californium sprite a lil bit. Modified prototype definitions as necessary. My discord is God of HyperDeath#6035, if you'd like to go into further discussion about sprite creation.

wretlaw120 committed:
Remade some sprites

Made new sprites for the three lowest tier of nuclear artillery shell (20t, 500t, and 1kt) and all nuclear rockets. Cleaned up californium sprite a lil bit. Modified prototype definitions as necessary.

wretlaw120 committed:
Fixed mipmap numbers

More-Wrong commented:
Before I merge this, would it be alright if you change the 15kt and 100kt artillery shells to match (just if the shade of green is different it looks a bit weird). The new sprites are great though!

wretlaw120 committed:
More sprites!

Updated remaining nuclear arty shells, and the thermobaric. Fixed spelling error in tritium processing tech and gave that one a new tech sprite, along with changing up the tritium item sprite to be a little cleaner. Updated all sprites that used the tritium item. Updated the thermobaric tech to have the new thermobaric arty shell. Also made californium tech and recipe use u-235 to better reflect the recipe. Moved 4t and 8t rockets behind scary atomic weapons to match the other californium-based weaponry. I want to change the tech sprites that are just explosions, but I don't have any good ideas on how to do that right now.
